libs/core/src/lib/timeline/components/timeline-node/timeline-node.component.html

```diff
@@ -0,0 +1,10 @@
+<div class="fd-timeline__node-container">
+    <div *ngIf="!glyph" class="fd-timeline__connecting-line"></div>
+    <div class="fd-timeline__node" [class.fd-timeline__node--icon]="!!glyph">
+        <fd-icon *ngIf="glyph" [glyph]="glyph"></fd-icon>
+    </div>
+    <div class="fd-timeline__connecting-line"></div>
+</div>
+<div tabindex="0" role="timelineitem" arial-label="timelineitem" class="fd-timeline__post fd-timeline__post--left">
```

Comment:
Missing i18n for aria-label.

Reply:
Could you please recheck this one?

Reply:
Sorry for the misinformation, as we're not using i18n in Core it should be refactoring using inputs.